packages/fields/src/fields/template/index.ts

```diff
@@ -14,9 +14,11 @@ const templateField: Field< BasePost > = {
 	id: 'template',
 	type: 'text',
 	label: __( 'Template' ),
-	getValue: ( { item } ) => item.template,
```

Comment:
This is not necessary. I removed it. https://github.com/WordPress/gutenberg/pull/66591#discussion_r1869465438